backend/src/background-jobs/cron-jobs.module.ts

```diff
@@ -8,7 +8,7 @@ export class BackgroundJobsModule {
   static withConfig(dbConnection: string): DynamicModule {
     return {
       module: BackgroundJobsModule,
-      imports: [ScheduleModule.forRoot(), ServicesModule.withDbonnection(dbConnection)],
+      imports: [ScheduleModule.forRoot(), ServicesModule.withDbconnection(dbConnection)],
```

Comment:
please rename it to withDbConnection to follow [camelCase](https://en.wikipedia.org/wiki/Camel_case)